pkg/cloudprovider/providers/azure/azure_standard.go

```diff
@@ -110,12 +110,6 @@ func (az *Cloud) getLoadBalancerProbeID(lbName, lbRuleName string) string {
 }
 
 func (az *Cloud) mapLoadBalancerNameToVMSet(lbName string, clusterName string) (vmSetName string) {
-	// Backends of Standard load balancer could belong to multiple VMAS or VMSS.
-	// Return "" to indicate so that following logic won't check this.
-	if az.useStandardLoadBalancer() {
-		return ""
-	}
-
```

Comment:
This will affect EnsureHostsInPool for availabilitySet if standard loadbalancer is on?

Reply:
@karataliu Yep, so availabilitySet logic is also updated [here](https://github.com/kubernetes/kubernetes/pull/62707/files#diff-5f68a21f3144ba02fbb07612bad83248R578)